### PR TITLE
[IPPInAppFeedback] UI banner component design tweaks

### LINKED ISSUE #8674: [IPPInAppFeedback] Polish UI banner component design
There are a few issues we should fix with the current banner design, as is using an existing component we didn't go through the differences with the design proposal:

- Text alignment:
> the banner message -"Rate your In-Person Payments experience"- does not go under the x button in the [original design](https://github.com/woocommerce/woocommerce-ios/pull/T16Ov1hzHWLJSOr8iN1BgR-fi-202%3A8998&t=Jr8EhOCOFMc4SeQm-0),
- Top separator
- “Share feedback” link color
- "Message" icon is slightly different

### PR DESCRIPTION
**Pull request by iamgabrielma**
Part of https://github.com/woocommerce/woocommerce-ios/issues/8530
Closes: #8674 

## Description
This PR updates the  In-Person Payments Feedback banner UI design to make it closer to the [design proposal](https://www.figma.com/file/T16Ov1hzHWLJSOr8iN1BgR/Feature-feedback?node-id=1%3A142&t=HXtzjijE8IxNLnWx-0).

## Changes
- Adds a new `commentContent` icon
- Overrides the default `.purple` color for the Action links with `.pink` (confirmed with design at C0354HSNUJH/p1674112248357019-slack-C0354HSNUJH)
- Appends a new top separator to the component by calling the existing `createBorderView()` in the main `UIStackView`

@toupper as [you mentioned in the prior PR](https://github.com/woocommerce/woocommerce-ios/pull/8640#issuecomment-1383809981), the banner text doesn't go below the `X` icon for dismissal, but I haven't found a way to make this happen with the current implementation without having to modify it entirely. Happy to hear if you have any pointer or recommendation! 🙇 

## Testing instructions
On a store with COD enabled, and set to US or CA:
1. Navigate to Orders, and see the In-Person Payments Feedback banner 
2. Confirm the icon, and color are correct as per the screenshot below:

| Design proposal | Current design |
|--|--|
| Screenshot 2023-01-23 at 15 30 37 |  |

### CODE REVIEW

File: WooCommerce/Classes/Extensions/UIImage+Woo.swift [hunk 1]
```diff
@@ -1174,6 +1180,7 @@ extension UIImage {
     static var builtInReaderError: UIImage {
         return UIImage(named: "built-in-reader-error")!
     }
+
```
Comment (toupper) [inline WooCommerce/Classes/Extensions/UIImage+Woo.swift]: Not a blocker, empty line here

File: WooCommerce/Resources/Images.xcassets/icon-comment-content.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
```
Comment (toupper) [inline WooCommerce/Resources/Images.xcassets/icon-comment-content.imageset/Contents.json:3]: Perhaps you can apply here a single scale with the pdf file

**Comment by iamgabrielma** [comment]
👋  Looping you in the PR as well @vanessamaartinstt . There's a known UI issue that we're discussing at p1674112248357019-slack-C0354HSNUJH :
Screenshot 2023-01-25 at 11 42 47

Edit: This is resolved.

File: WooCommerce/Classes/Extensions/UIImage+Woo.swift (hunk 1 above)
Comment (iamgabrielma) [reply WooCommerce/Classes/Extensions/UIImage+Woo.swift]: Fixed https://github.com/woocommerce/woocommerce-ios/pull/8728/commits/3993a92f842bf44680a0a96c3e2df39f427dc002

File: WooCommerce/Resources/Images.xcassets/icon-comment-content.imageset/Contents.json (hunk 2 above)
Comment (iamgabrielma) [reply WooCommerce/Resources/Images.xcassets/icon-comment-content.imageset/Contents.json:3]: Right! Switched from SVG to PDF and universal size here: https://github.com/woocommerce/woocommerce-ios/pull/8728/commits/26d3d0668b56c555c191dbad0d5e472baf09a64e

**Comment by iamgabrielma** [comment]
Resolved the issues, as well as the text moving below the dismiss button: 

We were using an `UILabel` there, which I switched with a new `StackView` that center the label between two empty Views. I opted to add the same right space as the dismiss button size (24px), but can be increased further if needed.

Ready for re-review @toupper 🙇

File: WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift [hunk 3]
```diff
@@ -167,6 +176,18 @@ private extension TopBannerView {
         return iconInformationStackView
     }
 
+    func createLabelHolderStackView() -> UIStackView {
+        labelHolderStackView.addArrangedSubviews([
+            createSeparatorView(height: 48, width: 0),
```
Comment (toupper) [inline WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift]: Could we add these values into constants? That way we can reuse them, and we know what they stand for

Comment (iamgabrielma) [reply WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift]: Definitely! https://github.com/woocommerce/woocommerce-ios/pull/8728/commits/4579e44d607237413e6b1fb4dd4e5a5c87a70e10

**Review by toupper — APPROVED** [review]

**Comment by toupper** [comment]
Great job @iamgabrielma ! Approved 🚢 :shipit:

**Pull request merged**